src/main/java/com/google/api/codegen/transformer/java/JavaSurfaceNamer.java

```diff
@@ -196,4 +198,10 @@ public String getReleaseAnnotation(ReleaseLevel releaseLevel) {
   public String getBatchingDescriptorConstName(Method method) {
     return inittedConstantName(Name.upperCamel(method.getSimpleName()).join("batching_desc"));
   }
+
+  @Override
+  public String getArtifactId(Model model) {
+    List<String> nameTokens = Splitter.on(".").splitToList(model.getServiceConfig().getName());
```

Comment:
Is this the standard way of getting the API name in toolkit? It would be nice if we had access to the `api_name` in the artman config: https://github.com/googleapis/googleapis/blob/master/gapic/api/artman_pubsub.yaml#L2